data-access/src/object_store/local.rs

```diff
@@ -230,4 +230,30 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn test_globbing() -> Result<()> {
```

Comment:
Might want a test somewhere of globstar (i.e. **)

Reply:
Done